src/libs/tools/lighttable.c

```diff
@@ -429,6 +518,23 @@ void gui_init(dt_lib_module_t *self)
   _lib_lighttable_zoom_slider_changed(GTK_RANGE(d->zoom), self); // the slider defaults to 1 and GTK doesn't
                                                                  // fire a value-changed signal when setting
                                                                  // it to 1 => empty text box
+
+  gtk_widget_set_sensitive(d->zoom_entry, (d->layout != DT_LIGHTTABLE_LAYOUT_CULLING_DYNAMIC && !d->fullpreview));
+  gtk_widget_set_sensitive(d->zoom, (d->layout != DT_LIGHTTABLE_LAYOUT_CULLING_DYNAMIC && !d->fullpreview));
+
+  d->layout_culling_restricted = dtgtk_togglebutton_new(dtgtk_cairo_paint_lt_mode_culling_restricted, 0, NULL);
```

Comment:
Gives an error `Gtk-CRITICAL **: 22:14:32.005: gtk_widget_set_visible: assertion 'GTK_IS_WIDGET (widget)' failed
` because the definition of the widget comes too late in the code : after the call to `_lib_lighttable_update_btn(self);`